presto-elasticsearch/src/main/java/io/prestosql/elasticsearch/ElasticsearchMetadata.java

```diff
@@ -158,9 +158,9 @@ private InternalTableMetadata makeInternalTableMetadata(String schema, String ta
     {
         ImmutableList.Builder<ColumnMetadata> result = ImmutableList.builder();
 
-        result.add(BuiltinColumns.ID.getMetadata());
```

Comment:
Since this fixes code that was introduced in the previous commit, squash it with that commit.